container-host-files/etc/scf/config/opinions-common.yml

```diff
@@ -189,6 +189,7 @@ properties:
       blobstore_type: webdav
       webdav_config:
         username: 'blobstore_user'
+    external_port: 9024
```

Comment:
You need to rebase your PR. This file doesn't exist anymore. It got renamed.

Reply:
At the moment I am sitting before the sle changes, yes. Going over that bump is low prio right now.

Reply:
I don't think it is an option. The merge is blocked due to this conflict. Just rebase and let CI run.

Reply:
When I have something working I will do the rebase. Right now I am not willing to bring the sle stuff into my checkout, as another variable which can break stuff, and confuse me.